packages/mux-player/src/themes/classic/index.ts

```diff
@@ -0,0 +1,17 @@
+// @ts-ignore
+import theme from './classic.html';
+import { document } from '../../polyfills';
```

Comment:
Should use the `globalThis` from the polyfills too.
```suggestion
import { document, globalThis } from '../../polyfills';
```